M2/Macaulay2/m2/latex.m2

```diff
@@ -72,20 +79,84 @@ sectionType = sectionNumber -> (
     "\\subsubsubsubsubparagraph");
 
 -----------------------------------------------------------------------------
+keywordTexMath = applyKeys(hashTable { -- both unary and binary keywords
+    symbol |- => "\\vdash ",
```

Comment:
Two comments:
1. The indenting is incorrect with the `applyKeys(..., symbolBody)` addition. Could you do that on the next line perhaps? (e.g. rewrite it with `keywordTexMath = applyKeys(keywordTexMath, symbolBody)`)
2. On that note, could you also adjust the entries so the `=>` are all aligned?

Reply:
what do you mean, the indenting is incorrect? do we have rules for indentation laid out somewhere?

Reply:
[Yes](https://github.com/Macaulay2/M2-emacs/blob/90ce16a15ea1f4fad08e8ba0b74b5045f4a72e8f/M2.el#L39-L42), four spaces for each unmatched parenthesis, etc. (and ideally tabs for each eight spaces, but this is more aesthetic).